simpleclient_jetty/src/main/java/io/prometheus/client/jetty/JettyStatisticsCollector.java

```diff
@@ -1,18 +1,17 @@
 package io.prometheus.client.jetty;
```

Comment:
I don't think that helps much, we going to run into some classes which work on 1.6 and other than need 1.8 so we'd likely just end up with jetty_stats and jetty_stats_jdk8.